org-code-javabuilder/protocol/src/main/java/org/code/protocol/AssetFileHelper.java

```diff
@@ -63,6 +95,23 @@ private void loadStarterAssetsList() {
     }
   }
 
+  private void loadUserAssetsList() {
+    try {
+      final JSONArray jsonResponse =
+          new JSONArray(getRequest(this.generateUserAssetsListUrl(), this.httpClient));
+      this.userAssetFilenames =
+          jsonResponse
```

Comment:
if there are no assets will this set userAssetFilenames to an empty list or throw an exception?

Reply:
Good question - it should return an empty list so we should not hit a JSON exception here. I'll add a test case to validate.